sdk/eventhub/Azure.Messaging.EventHubs/assets/live-tests-azure-setup.ps1

```diff
@@ -56,6 +56,8 @@ Import-Module Az.Resources
 # == Function Definitions ==
 # ==========================
 
+. .\functions\functions.ps1

```

Comment:
We should probably give this file a more descriptive name.   Maybe something like `azure-principal-functions` or similar?